influxdata/tokenizer.go

```diff
@@ -111,24 +213,28 @@ func (t *Tokenizer) readMore() {
 	n := cap(t.buf) - len(t.buf)
```

Comment:
this comment is for line `109  211`:

The return for if t.complete is already true isn't covered by tests.

Reply:
yes, that's because `readMore` is only called by `ensure`, and `ensure` checks `t.complete` just before calling it, so the check in this function is just for completeness (the condition can never be met in practice). I don't think it's worth adding a direct internal test directly to `readMore` just for this, as the logic of that condition is simple enough to be obviously correct IMHO.